.github/workflows/kind.yml

```diff
@@ -238,6 +238,7 @@ jobs:
             --coverage \
             --feature-gates AllAlpha=true,AllBeta=true \
             --proxy-all \
+            --no-kube-proxy \
```

Comment:
I think "test-e2e-encap-non-default" covers it?

But I feel it may be more proper to have "--no-kube-proxy" in that job which is dedicated to test specific scenario that can't be covered by other jobs. This job is dedicated to test all features. Since the feature works on clusters with kube-proxy, it shouldn't add the varient.